Quadriga/src/main/webapp/WEB-INF/tiles/layouts/skeleton_public.jsp

```diff
@@ -46,6 +46,7 @@
           </div>
           <div id="navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav">
+              <tiles:insertAttribute name="navigationSites" />  
```

Comment:
remove this and instead create a new skeleton that uses an attribute where it currently prints the project name. then set that attribute in your tiles def.